contract/tests/test_contract.py

```diff
@@ -1739,7 +1742,11 @@ def test_renew_create_new_line(self):
         self.assertFalse(self.acct_line.is_auto_renew)
         self.assertTrue(new_line.is_auto_renew)
         self.assertEqual(new_line.date_start, date_start + relativedelta(months=12))
-        self.assertEqual(new_line.date_end, date_end + relativedelta(months=12))
+        # Be sure that the date is correct (leap years)
+        self.assertEqual(
+            new_line.date_end,
+            date_start + relativedelta(months=24) - relativedelta(days=1),
```

Comment:
Why 2 years ?

Reply:
And why they aren't fixed dates (example, '2022-01-01')? It's one of the good test practices.